### PR TITLE
Manifest fixes

### PR DESCRIPTION
**Pull request by fregante**
See review

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,10 +1,15 @@
 {
   "manifest_version": 2,
-  "author": "Pixie Brix, LLC",
```
Comment (fregante) [inline src/manifest.json:3]: The author doesn't seem to appear anywhere: https://chrome.google.com/webstore/detail/pixiebrix/mpjjildhmpddojocokjkgmlkkkfjnepo

And also is outdated and redundant IMHO

File: src/manifest.json [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
   "name": "PixieBrix",
-  "short_name": "PixieBrix",
```
Comment (fregante) [inline src/manifest.json:5]: Redundant

File: src/manifest.json [hunk 3]
```diff
@@ -1,10 +1,15 @@
...
-  "description": "PixieBrix - integrate your favorite web applications without code",
+  "description": "Integrate and extend your favorite SaaS apps without writing a line of code!",
```
Comment (fregante) [inline src/manifest.json:4]: The description appears in 2 places:


- In the first shot, the name is duplicate.
- In the second shot, the intro is duplicate/slightly different.

Screen Shot 8

Screen Shot 6



## After

Screen Shot 7

Screen Shot


## Todo: @twschiller 

- [ ] Drop first line of the description in the Chrome Web Store

File: src/manifest.json [hunk 4]
```diff
@@ -1,10 +1,15 @@
...
+  "homepage_url": "https://www.pixiebrix.com",
```
Comment (fregante) [inline src/manifest.json:5]: This link appears in the browser action dropdown:

## Before

Screen Shot 1


## After

Screen Shot 5

File: src/manifest.json [hunk 5]
```diff
@@ -1,10 +1,15 @@
...
   "minimum_chrome_version": "80.0",
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "extension@pixiebrix.com",
```
Comment (fregante) [inline src/manifest.json:9]: This `id` is made up. If you never uploaded the extension before, we can use this. If there's an extension already there, we must to use the ID you can find on AMO (Mozilla Addons)

File: src/manifest.json [hunk 6]
```diff
@@ -1,10 +1,15 @@
...
+      "strict_min_version": "72.0"
```
Comment (fregante) [inline src/manifest.json:10]: Firefox ESR for now, I don’t think it actually works though

File: src/manifest.json (hunk 1 above)
Comment (twschiller) [reply src/manifest.json:3]: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/author

image

It looks like we should leave it in, and update the entity name ("PixieBrix, Inc.")

I suspect have a company name in there may also help with review times

File: src/manifest.json (hunk 2 above)
Comment (twschiller) [reply src/manifest.json:5]: It's not redundant, as when the name is changed in the development version "PixieBrix - Development", the name is longer than the recommended name

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/short_name

File: src/manifest.json (hunk 5 above)
Comment (twschiller) [reply src/manifest.json:9]: > This `id` is made up. If you never uploaded the extension before, we can use this. If there's an extension already there, we must to use the ID you can find on AMO (Mozilla Addons)
> 

I'll probably want to check, as IIRC I made it some distance uploading an earlier version to the AMO in the past

@twschiller [ ] check if we have an email address already associated with our extension in AMO

File: src/manifest.json (hunk 6 above)
Comment (twschiller) [reply src/manifest.json:10]: We're OK just supporting the last 2 versions

**Review by twschiller — CHANGES_REQUESTED** [review]
See individual comments

File: src/manifest.json (hunk 3 above)
Comment (twschiller) [reply src/manifest.json:4]: I'll tweak the wording a little bit on the description before I merge in the code after I re-review

File: src/manifest.json (hunk 5 above)
Comment (fregante) [reply src/manifest.json:9]: It’s not supposed to be email, this is akin to `com.apple.QuickTime`, which looks like a reversed URL but doesn’t exist.

But if the extension was already uploaded and you want to reuse that then we’ll have to change it to the Mozilla-provided key

File: src/manifest.json (hunk 2 above)
Comment (fregante) [reply src/manifest.json:5]: Gotcha I didn’t think you’d want to ship alternative channels with longer names.

File: src/manifest.json (hunk 6 above)
Comment (fregante) [reply src/manifest.json]: I updated it to v89, which is what I've been testing it with. v90 is already out

**Review by twschiller — APPROVED** [review]
image

**Pull request merged**